app/src/main/java/com/codepath/travel/adapters/viewholders/SuggestedPlaceViewHolder.java

```diff
@@ -37,9 +37,9 @@ public void populate(SuggestionPlace suggestionPlace) {
             rbPlaceRating.setRating(suggestionPlace.getRating().floatValue());
         }
         if (suggestionPlace.isSelected()) {
-            ivAddSuggestionPlace.setImageResource(android.R.drawable.star_big_on);
```

Comment:
Or if you prefer the add and tick icons, you can rename checkbox_star to checkbox_add or something and then replace the references to use the add/tick icons. That way it will appear consistent on the suggestion view and in the detail view.